distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -436,6 +436,7 @@ The Apache Software License, Version 2.0
     - io.grpc-grpc-rls-1.55.3.jar
     - io.grpc-grpc-servlet-1.55.3.jar
     - io.grpc-grpc-servlet-jakarta-1.55.3.jar
+    - io.grpc-grpc-util-1.60.0.jar
```

Comment:
Mixing grpc library versions doesn't look good. grpc version is currently coupled to protobuf version and also to bookkeeper version so we cannot safely upgrade protobuf. (see thread https://lists.apache.org/thread/s9g6w31vtwzgqf162hhlcr2nx3y68gv5)